elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/data/value/ElasticsearchExprValueFactory.java

```diff
@@ -177,14 +183,22 @@ public ExprValue construct(String field, Object value) {
       return constructString((String) value);
     } else if (type.equals(BOOLEAN)) {
       return constructBoolean((Boolean) value);
-    } else if (type.equals(TIMESTAMP)) {
+    } else if (type.equals(TIMESTAMP) || type.equals(DATE) || type.equals(TIME)
```

Comment:
Quesiton,  In which case the field will in type of DATA/TIME?
The main purpose of ExprValueFactory is extract data from Elasticsearch response. Elasticsearch only support date type and it is mapped to TIMESTAMP.

Reply:
Never mind, this is the aggregation result. I see the problem.